element_deeplabcut/dlc.py

```diff
@@ -511,12 +606,12 @@ def make(self, key):
     @classmethod
     def get_trajectory(cls, key, body_parts='all'):

```

Comment:
I'm wondering if we need this method at all. This is a convenient method for data extraction from the pipeline?